src/components/bulkConcepts/component/Pagination.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  setNextPage,
+  setPreviousPage,
+} from '../../../redux/actions/concepts/addBulkConcepts';
+
+
+export class Paginations extends Component {
+  static propTypes = {
+    firstDictionaryIndex: PropTypes.number.isRequired,
+    dictionaries: PropTypes.number.isRequired,
+    currentPage: PropTypes.number,
+    lastPage: PropTypes.number,
+    setNextPage: PropTypes.func.isRequired,
+    setPreviousPage: PropTypes.func.isRequired,
+    view: PropTypes.bool.isRequired,
+  };
+
+  static defaultProps = {
+    currentPage: 1,
+    lastPage: 2,
+  }
+
+  render() {
+    const {
+      firstDictionaryIndex, dictionaries, lastPage, currentPage,
+      setNextPage: NextPage, setPreviousPage: PreviousPage, view,
+    } = this.props;
+    return (
+      <Fragment>
+        {view && (
+        <div className="container">
+          <div className="row">
+            <div className="col-md-6 text-left">
+              <i>
+                Showing
+                {' '}
```

Comment:
Shouldn't you be using the html non breaking space here?

Reply:
I used it because it is within a jsx file